### ISSUE TITLE
Pickups move erratically, hide inside floor

### ISSUE THREAD

**Issue by spekdrum** [cuberite/cuberite#3729]
Client version:   1.11.2
Server OS:        Debian
Commit id:        84bdba

### Expected behavior
Pickups falling on the floor should just rest on it...

### Steps to reproduce the behavior
...but frequently happens that they tend to rest on the block below, trespassing the one where they should lay on. Apparently the pickup becomes invisible but if you look carefully when a block becomes a pickup, it just falls and "sinks" in the top block until it falls to the underlying block. 
If you destroy the block where the  drop is hidden you will see that both (the new one and the old one) drops gets together and sink again.

At the very bottom of a chunk if you try to replicate this behavior, you will see drops dancing, like if they couldn't find a stable floor to rest on. They sink down a bit but they suddenly go up, over and over again.

I didn't have a look at the code yet, but it looks like the typical collision detection problem usually found in games in development.

(edited by madmaxoft to fix naming conventions / typos)

**Comment by madmaxoft** [comment]
Most likely fixed in #3720.

**Comment by spekdrum** [comment]
I did check out on FixEntity branch, did several tests with the same results, sorry.

**Comment by spekdrum** [comment]
When I submited this issue I actually was working in a modified master branch, where all the changes made for entity tracer replacement were removed (my server is running with this one). This behavior is present in **FixEntityTracer** branch, but is not in the **master**.  This led me to confussion.
I found a way to solve this problems but in my modified branch. So those fixes I made are not 100% compatible with the tracer replacement changes (damn xD).
I will better stop fighting with this 'till those fixes on entity tracer are resolved. IMHO it would be better that those changes on tracer get fixed soon because currently master in unplayable.
